src/nvim/mouse.c

```diff
@@ -844,13 +836,10 @@ bool do_mouse(oparg_T *oap, int c, int dir, int count, bool fixindent)
     }
 
     if (fixindent) {
-      c1 = (dir == BACKWARD) ? '[' : ']';
-      c2 = 'p';
+      prep_redo(regname, count, NUL, (dir == BACKWARD) ? '[' : ']', NUL, 'p', NUL);
     } else {
-      c1 = (dir == FORWARD) ? 'p' : 'P';
-      c2 = NUL;
+      prep_redo(regname, count, NUL, (dir == FORWARD) ? 'p' : 'P', NUL, NUL, NUL);
     }
-    prep_redo(regname, count, NUL, c1, NUL, c2, NUL);
```

Comment:
Why remove the `c1` and `c2` variables? That makes the code harder to understand.